src/MLOps.NET.SQLite/Storage/SQLiteMetaDataStore.cs

```diff
@@ -38,6 +41,15 @@ public async Task<Guid> CreateRunAsync(Guid experimentId, string gitCommitHash =
             }
         }
 
+        ///<inheritdoc/>
+        public IConfusionMatrixEntity GetConfusionMatrix(Guid runId)
+        {
+            using (var db = new LocalDbContext())
+            {
+                return db.ConfusionMatrices.Single(x => x.RunId == runId);
```

Comment:
Not every run will have stored a confusion matrix I think. Given that, I think it may make sense to instead `.SingleOrDefault` here so it doesn't blow up